src/grids/css/grids-core.css

```diff
@@ -55,12 +48,9 @@
 
 .pure-u {
     display: inline-block;
-    *display: inline; /* IE < 8: fake inline-block */
-    zoom: 1;
     letter-spacing: normal;
     word-spacing: normal;
-    vertical-align: top;
-    text-rendering: auto;
+    vertical-align: bottom;
```

Comment:
This is #539 (which I replicated in testing) & removing text-rendering as text-rendering isn't W3C valid, even at CSS3 spec.